### PR TITLE
Initial Example Code

### PR DESCRIPTION
**Pull request by AllenVan**
Raul please

### CODE REVIEW

**Comment by RaulCard** [comment]
write a better title for the PR

**Review by RaulCard — CHANGES_REQUESTED** [review]
Add missing steps to the README for setting up `tutorial-env`. Add scripts dir to gitignore so these environment files are not included.

File: example_bot.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+async def on_ready():
+    await tree.sync(guild=discord.Object(id=688838108749365350))
+    print("Ready!")
+
+
+#### Create the initial embed object ####
+embed=discord.Embed(title="Example", url="https://i.redd.it/4rb49b4ot6ra1.png", description="Sample Embed", color=0x109319)
+
+# Add author, thumbnail, fields, and footer to the embed
+embed.set_author(name="RaulCard", url="https://github.com/RaulCard", icon_url="https://avatars.githubusercontent.com/u/31727722?v=4")
```
Comment (RaulCard) [inline example_bot.py:39]: why my name here 💀

Comment (AllenVan) [reply example_bot.py:39]: If you don't like it make a new pull request fixing it

**Comment by AllenVan** [comment]
-Removed all environment directories
-Updated ReadMe to have comprehensive instructions to create virtual environment
-Added bot-env/ directory to .gitignore

File: example_bot.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+intents = discord.Intents.default()
+intents.message_content = True
+client = discord.Client(intents=intents)
+tree = app_commands.CommandTree(client)
+
+
+@client.event
+async def on_ready():
```
Comment (RaulCard) [inline example_bot.py:14]: This is overwritten. Combine the two `on_ready()` functions or delete one.

File: example_bot.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    print(f'We have logged in as {client.user}')
+
+@client.event
+async def on_message(message):
```
Comment (RaulCard) [inline example_bot.py:18]: It aint working for me. Maybe there is an intent we need.

**Review by RaulCard — CHANGES_REQUESTED** [review]

File: example_bot.py (hunk 3 above)
Comment (RaulCard) [reply example_bot.py:18]: Ah, bro you overwrite this function too

**Pull request closed without merging**